packages/testlab/package.json

```diff
@@ -39,10 +41,10 @@
     "README.md",
     "index.js",
     "index.d.ts",
-    "dist/src",
-    "dist/index*",
+    "dist",
     "should-as-function.d.ts",
-    "src"
+    "src",
+    "!*/__tests__"
```

Comment:
This excludes both `src/__tests__` and `dist/__tests__` from the npm package tarball.